oak_runtime/src/lib.rs

```diff
@@ -242,6 +242,38 @@ impl NodePrivilege {
             can_endorse_integrity_tags: top_tag,
         }
     }
+
+    /// Generates a new [`Label`] from `label` that is downgraded as much as possible using the
+    /// current privilege.
+    fn downgrade_label(&self, label: &Label) -> Label {
+        let has_top_privilege = self.can_declassify_confidentiality_tags.contains(&top());
```

Comment:
I know this was pre-existing, but could you add some unit tests for this function? Maybe in a separate PR if you prefer.

Reply:
Added unit tests.